src/StyledStrings.jl

```diff
@@ -16,12 +16,29 @@ include("legacy.jl")
 
 using .StyledMarkup
 
-function __init__()
+const HAVE_LOADED_CUSTOMISATIONS = Ref(false)
+
+"""
+    load_customisations!(; force::Bool=false)
+
+Load customisations from the user's `faces.toml` file, if it exists as well as
+the current environment.
+
+This function should be called before producing any output in situations where
+the user's customisations should be considered.
+
+Unless `force` is set, customisations are only applied when this function is
+called for the first time, and subsequent calls are a no-op.
+"""
+function load_customisations!(; force::Bool=false)
+    !force && HAVE_LOADED_CUSTOMISATIONS[] && return
```

Comment:
This should probably be an atomic for thread safety?

Reply:
Maybe? Aside from suspected unlikeliness, without change to the underlying files/environment `load_usercustomisations!` should be idempotent. Furthermore the actual modifications are made (by `loadface!`) to a `ScopedValue`, which IIRC is already threadsafe?

In other words, it seems to me that in the unlikely event a threading issue/race condition was run into here, it wouldn't actually have any impact.

Reply:
That means that you don't need a lock, but you still need the write `HAVE_LOADED_CUSTOMISATIONS[] = true` to be atomic or the read `HAVE_LOADED_CUSTOMISATIONS[]` is not guaranteed to give you `true` or `false`

Reply:
Ah funky, thanks for explaining.